src/V3Task.cpp

```diff
@@ -1379,6 +1379,7 @@ class TaskVisitor final : public VNVisitor {
             m_insStmtp = nullptr;
             m_modNCalls = 0;
             iterateChildren(nodep);
+            if (AstClass* classp = VN_CAST(nodep, Class)) classp->repairCache();
```

Comment:
Should instead V3Task set a bool when it does a function update, then only call this when it needs to?

Reply:
Good idea. I might just add it.